lib/features/support/internal_hooks.rb

```diff
@@ -109,6 +109,7 @@
     output_received_requests('sessions')
     output_received_requests('builds')
     output_received_requests('logs')
+    output_received_requests('invalid requests')
```

Comment:
If there were no invalid requests, the message to users will be quite confusing "No valid invalid requests received".  As we're now logging invalid requests, perhaps we can just remove the word "valid" from the message.